storage/innobase/handler/ha_innodb.cc

```diff
@@ -984,6 +990,10 @@ static SHOW_VAR innodb_status_variables[]= {
   (char*) &export_vars.innodb_buffer_pool_wait_free,	  SHOW_LONG},
   {"buffer_pool_write_requests",
   (char*) &export_vars.innodb_buffer_pool_write_requests, SHOW_LONG},
+  {"checkpoint_age",
+  (char*) &export_vars.innodb_checkpoint_age,		  SHOW_LONG},
+  {"checkpoint_max_age",
+  (char*) &export_vars.innodb_checkpoint_max_age,	  SHOW_LONG},
   {"data_fsyncs",
   (char*) &export_vars.innodb_data_fsyncs,		  SHOW_LONG},
   {"data_pending_fsyncs",
```

Comment:
We do not currently seem to count the `io_submit()` calls, but that can be added if it is considered useful.
```c
  {"buffered_aio_submitted",
  (char*) &export_vars.innodb_buffered_aio_submitted,	  SHOW_LONG},
```